lib/rootstrap/support/Rakefile

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+task :code_analysis do
+  sh 'bundle exec rubocop lib spec'
+  sh 'bundle exec reek lib'
```

Comment:
don't we need to run ```rails_best_practices```?

Reply:
I don't think we do because the output project won't be a rails one but an empty gem